CRM/Membershipextrasdefaultconfig/Installationsteps/SampleContributionPage.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+/**
+ * Class to create sample contribution page for membership extras.
+ */
+
+class CRM_Membershipextrasdefaultconfig_Installationsteps_SampleContributionPage {
+
+  /**
+   * Main method that will apply all changes.
+   * This is called inside public function install of
+   * class CRM_Membershipextrasdefaultconfig_Upgrader
+   */
+  public function apply() {
+    civicrm_api3('ContributionPage', 'create', [
+      'title' => "Compuclient default contribution page",
```

Comment:
use the API first to check that there is no any contribution page with the name "Compuclient default contribution page" before creating it